docs/architecture.md

```diff
@@ -296,14 +296,14 @@ that routes Pod traffic directly to the destination Node without going through
 the extra hop of the Node network router. Antrea Agent also creates the iptables
 (MASQUERADE) rule for SNAT of Pod-to-external traffic.
 
-[Antrea supports GKE](/docs/gke-installation.md) with `NoEncap` mode.
+[Antrea supports GKE](gke-installation.md) with `NoEncap` mode.
 
 * ***NetworkPolicyOnly*** Inter-Node Pod traffic is neither tunneled nor routed
 by Antrea. Antrea just implements NetworkPolicies for Pod traffic, but relies on
 another cloud CNI and cloud network to implement Pod IPAM and cross-Node traffic
-forwarding. Refer to the [NetworkPolicyOnly mode design doc](docs/policy-only.md)
+forwarding. Refer to the [NetworkPolicyOnly mode design doc](policy-only.md)
 for more information.
 
-[Antrea for AKS Engine](https://github.com/Azure/aks-engine/blob/master/docs/topics/features.md#feat-antrea)
-and [Antrea EKS support](/docs/eks-installation.md) work in `NetworkPolicyOnly`
-mode.
+[Antrea for AKS
```

Comment:
Any specific reason to break the line here?

Reply:
I think that's how my editor breaks line automatically to stay within 80 chars. It will only break lines at whitespaces. Let me check if it screws up formatting.